src/api/plans.ts

```diff
@@ -52,14 +52,14 @@ router.get("/plans", async (req, res) => {
 const planReqBody = z.object({
   graduationDate: z.string().date(),
   programId: z.number(),
-  concentrationId: z.number(),
+  concentrationId: z.number().nullable().optional(),
 });
 router.post("/plans", async (req, res) => {
   const user = await authUser(req);
   const planBody = planReqBody.parse(req.body);
 
   const planId = await one<{ id: number }>(
-    sql`INSERT INTO user_plans (user_id, graduation_date, concentration_id, program_id) VALUES (${user.id}, ${planBody.graduationDate}, ${planBody.concentrationId}, ${planBody.programId}) RETURNING id`,
+    sql`INSERT INTO user_plans (user_id, graduation_date, concentration_id, program_id) VALUES (${user.id}, ${planBody.graduationDate}, ${planBody.concentrationId ?? null}, ${planBody.programId}) RETURNING id`,
```

Comment:
this is awkward... please fix this formatting.

Reply:
Out of scope for this PR, existing issue